README.md

```diff
@@ -1 +1,43 @@
 # Restaurant-page
+
+This project project is part of Microverse learning and here I created dynamic restaurant website. This projet was created only with javascript and it does not need any kind of browser to render the pages.
```

Comment:
after the period

Reply:
Thank you for pointing this out again. I certainly forgot to change in the first time